src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -203,31 +219,43 @@ public AudioData generateAudioData(AudioData audioData, String toSpeak) throws I
     return audioData;
   }
 
+  public boolean setSsml(boolean ssml) {
+    PollyConfig c = (PollyConfig) config;
+    c.ssml = ssml;
+    return ssml;
+  }
+
+  public boolean setAutoDetectSsml(boolean ssml) {
+    PollyConfig c = (PollyConfig) config;

```

Comment:
this shouldn't reference config here.. please remove config as a member from these classes.

Reply:
The more variable you make the more error prone it becomes, if I was to pull it out of config then I'd still need to maintain it in config, and additionally copy it into then copy it out of that config.   Your recommending more maintenance in the future, which I am against.